.github/workflows/deploy.yml

```
@@ -0,0 +1,65 @@
+        required: true
+      IMAGES:
+        type: string
+        required: true
+      FEATURE_NAME:
+        type: string
+        required: true
+jobs:
+  deploy:
+    runs-on: arc-runners
+    env:
+      GIT_BRANCH_DEPLOY: ${{ inputs.outputs.GIT_BRANCH_DEPLOY }}
```

actionlint (1.7.4)

property "outputs" is not defined in object type {docker_tag: string; feature_name: string; git_branch_deploy: string; images: string}

(expression)